rllib/agents/es/es.py

```diff
@@ -228,10 +228,18 @@ def validate_config(self, config: TrainerConfigDict) -> None:
                 "`NoFilter` for ES!")
 
     @override(Trainer)
-    def _init(self, config, env_creator):
+    def setup(self, config):
+        # Call super's validation method.
         self.validate_config(config)
+
+        # Generate `self.env_creator` callable to create an env instance.
+        self._get_env_creator_from_env_id(self._env_id)
```

Comment:
could we change the name of this function to `_set_env_creator_from_env_id`. This function sets the `self.env_creator` variable, but it doesn't return anything. That, or we could return the env creator ourselves and set the attribute ourselves:

```
self.env_creator = self._get_env_creator_from_env_id(self._env_id)
```

Reply:
thanks for the catch. I'll check. ...

Reply:
done